recipes/optick/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import is_msvc_static_runtime, is_msvc
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rm, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+import os
+
+required_conan_version = ">=2.0.0"
+
+class PackageConan(ConanFile):
+    name = "optick"
+    description = "Optick is a super-lightweight C++ profiler for Games"
+    license = "MIT"
+    url = "https://github.com/bombomby/optick"
+    homepage = "https://github.com/bombomby/optick"
+    topics = ("profiler", "games")
+    package_type = "shared-library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {}
+    default_options = {}
+
+    @property
+    def _min_cppstd(self):
+        return 11
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        pass
+
+    def configure(self):
+        pass
+
```

Comment:
```suggestion
```

Not really required, as its a shared-only library and there are not patches to be applied.